popup.html

```diff
@@ -143,7 +143,8 @@
                     <div id="dropbox_risk">{{ i18n.dropbox_risk }}</div>
                     <label for="token_input">{{ i18n.dropbox_token }}</label>
                     <input type="text" id="token_input" v-model="dropboxToken" disabled>
```

Comment:
Action buttons are more important than the token field. Buttons should be at the top of the page since they are what the users will be interacting w/, so it might be a good idea to move this down two lines.